fastlane/Fastfile

```diff
@@ -19,7 +19,7 @@ platform :android do
 
   desc "Runs tests"
   lane :check do
-    gradle(tasks: ["check","koverXmlReportDebug"])
+    gradle(tasks: ["testDebug", "lintDebug", "detekt","koverXmlReportDebug"])
```

Comment:
I made a separate PR for this change:

https://github.com/bitwarden/authenticator-android/pull/242